compiler/hermes/include/hermes/core/MessageBuffer.h

```diff
@@ -33,14 +34,16 @@ namespace hermes
 class MessageBuffer final
 {
 public:
-  MessageBuffer(MessageBus *);
+  MessageBuffer(MessageBus *bus);
```

Comment:
I think this is not related with the purpose of this change.
Please change in another PR if you want to.